### PR TITLE
API side filter for public published documents

### PR DESCRIPTION
**Pull request by dhlevi**
Fix to the API side to prevent the public endpoint from showing unpublished documents

### CODE REVIEW

File: api/controllers/search.js [hunk 1]
```diff
@@ -834,8 +845,11 @@ var searchCollection = async function (roles, keywords, schemaName, pageNum, pag
     }
   });
 
+  console.log(JSON.stringify(aggregation));
```
Comment (KitArmstrong) [inline api/controllers/search.js]: Can the log be removes?

File: epic_resource_model [hunk 2]
```diff
@@ -0,0 +1 @@
...vTZM1XfIrwOorZO2Kslzk5WOUSrqsACqjPEYJyeRI7xQJLqPoK7jiKeiXoxdj5wlWLWmvzYnh7ZYggbcX6rogew3fYUEI8c9OauKSUKUmTgdVajjez0QcKZQ2DsKQEBc1iPm3bpKM8kyV3VUP/aBSRgPDW3KBdryIbUhA9Dowx5sy0MTfFeawqpSpeNpm/xIpwlvprRjgvafTPMlaPBYU5R1pvaT+Ds6mGdmf0ErDOxU/KQ03f5PebevQlmVX2btpvVJpYdNvC4t2Voy6jDPanlKmr8GqokxoiZ9KNFuvIePeU00n76gsRR7ruP6S5NEcvkCEkG5LbD9WMD58BWPXUTgBjRv+rjGn0+OV3FcyHLEVetsVP7YP32Q0mzQhRSulodIq9UNCMIsyyPcSshQspFWXa6vosuh1kIQ8tJjSBM1JFfHcaUYbq3bKxN/+SXka7W0Khf1He9vOM8wtcPyqSXmeyPDebXDj7NLUmMXbwPnGOF4nnGs9+iVmu+H7JVa2AfodZeTgK0gxBU9fJLKv4JP4DYB6A6QjnLAHSLe8MuaYHgBkHjoKWbl2qYjjhkbpVsukZ7e15B4g3ZN0pTvcwuf5bd4qhvYHLm67p/hhPQj5EiU13j8FiwUupRQmM7yJzaGmhsoTXSQrkjmOhoMrE8kKJInKFKGaR1HxG2Jm9KaDNfw5B/lsyelmidJscHklBLCLc6qRSpFgsGHkSzixC07LqgxTsA7hi8IVnmblc2yg3EjCU051pqoIlEFeC64FWVWJeorwO8aVp2gZqHYni6Q+tKok1Kf2AHgvjihMjdFGYYZLSfG1gd+nyJPn0DAruhs7nCpPFi8xiLEiDrggpuz2e/iQ/Gpz7FW9+wBh+b7iRmkJiiO8ktyjSTZHOs2NST6Vhb8fwN+qGoBqC7xirzXNbNjmEUzzPgkSR88a9FQrOWsINOXXYg0OzhpZQytrUKwrZQ1fEHnzWqzxJrKDjVPHdWp8hDIZ9oLlhzo4xFIG36LitYNnO7TDgJjgZvVsh/IkLWFfQ7nI/QmVeaU3vshrE4+OaWIVB913NKHpi4Immhzx0/eUxjw9qjTmKetGvUyDOeKKq9trBJYrDzxB4MbZ7TX8+TkJkUngkEi2Nln4ZCSlSZFWoGXDXk7awNYsifZFtvBjmiR5fT2lwWZJXq599V8=</diagram></mxfile>
```
Comment (KitArmstrong) [inline epic_resource_model]: Is this the data model diagram?

File: api/controllers/search.js (hunk 1 above)
Comment (dhlevi) [reply api/controllers/search.js]: Yes, forgot I left that in

File: epic_resource_model (hunk 2 above)
Comment (dhlevi) [reply epic_resource_model]: Yep. I moved that into the docs repo so it can be removed from this one

**Pull request merged**